flink-yarn-tests/src/test/java/org/apache/flink/yarn/YarnConfigurationITCase.java

```diff
@@ -91,11 +90,11 @@ public void testFlinkContainerMemory() throws Exception {
 
                     final YarnConfiguration yarnConfiguration = getYarnConfiguration();
                     final YarnClusterDescriptor clusterDescriptor =
-                            YarnTestUtils.createClusterDescriptorWithLogging(
```

Comment:
The logging part should be entirely ignored because no conf directory will be found.